### PR TITLE
task 6

### PR DESCRIPTION
**Pull request by Zoba1**
please review my work
@promise-J
@chidipatrick

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <br />
+            <label for="email">Email Address:</label>
+            <input type="email" id="email" name="email" required />
+            <br />
+            <label for="phone">Phone number:</label>
+            <input type="tel" id="phone" name="phone" required />
+            <br />
+            <label for="username">Username:</label>
+            <input type="text" id="username" name="username" required />
+
```
Comment (ChidiPatrick) [inline html/index.html:24]: watch you indentation

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <br />
+            <label for="password">Password:</label>
+            <input type="password" id="password" name="password" required />
+        </form>
+        <form action="#">
+            <label for="gender">Gender:</label>
+            <input type="radio" name="gender" id="gender" />
+            <label for="male">Male</label>
+            <input type="radio" name="gender" id="male">
```
Comment (ChidiPatrick) [inline html/index.html]: close your tag

File: html/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <select name="country" id="country">
+                <option value="select">Select Country</option>
+                <option value="India">India</option>
+                <option value="country">Select Country</option>
+                <option value="country">Select Country</option>
+                <option value="country">Select Country</option>
+            </select>
+            <br />
+            <label for="terms-and-conditions">Terms and condition</label>
+            <input type="checkbox" name="agree" id="terms-and-conditions">
```
Comment (ChidiPatrick) [inline html/index.html]: close your input tag

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Nice work close all your tags

File: html/index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Registration</title>
+        <link rel="icon" href="../img/download.jpg" />
+    </head>
+    <body>
+        <form action="#" method="post">
+            <h2>Registration Form</h2>
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="name" ></input>
```
Comment (ChidiPatrick) [inline html/index.html]: Input element is a self-closing tags like img element. Remove the closing tag you put here

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Close all you input elements properly

File: html/index.html (hunk 4 above)
Comment (Zoba1) [reply html/index.html]: @ChidiPatrick  ii have done the changes

**Review by ChidiPatrick — APPROVED** [review]
Great work

**Pull request merged**